content/en/docs/architecture/ci-operator.md

```diff
@@ -431,6 +432,13 @@ ci-cluster-pool   ci-ocp-4-7-amd64-aws-us-east-1   1       1      hive.aws.ci.op
 
 Note that `cluster_claim` and `cluster_profile` are mutually exclusive because [the latter](/docs/architecture/step-registry/#implicit-lease-configuration-with-cluster_profile) indicates installing a cluster on demand, instead of claiming a pre-installed cluster in a pool.
 
+If you are using `cluster_claim` to replace a workflow such as [ipi-aws](https://steps.ci.openshift.org/workflow/ipi-aws), note that you may need to include steps to
+- Configure RBAC for image pulls (pre)
+- Gather logs, must-gather, etc. (post)
+
+...as these are not otherwise performed. A [generic-claim workflow](https://steps.ci.openshift.org/workflow/generic-claim) is provided for exactly this purpose.
+You can include it under `test:` as shown in the example above.
+
```

Comment:
If `cluster_claim` is used to replace a workflow, e.g., [ipi-aws](https://steps.ci.openshift.org/workflow/ipi-aws), then as a result, the steps such as log gathering are also removed along with the workflow. They
can be retrieved by introducing a new workflow which removes only the creating and destroying steps in the workflow, e.g., [generic-claim workflow](https://steps.ci.openshift.org/workflow/generic-claim).

-----
We do not mention RBACs for pull because it will be handled by https://issues.redhat.com/browse/DPTP-2255

Reply:
Not sure what exactly you were suggesting needs to be changed here, but I gave it a shot.

Reply:
I did not know the step for exposing images has been fixed by Petr when I drafted the first review.
That is why I do not want to mention it at the begin.
And then add it back when I knew.

Sorry for the confusion.
All good now.